### ISSUE TITLE
After disabled and re-enabled the skill, no devices are found

### ISSUE THREAD

**Issue by woody4165** [hardillb/node-red-alexa-home-skill-web#76]
Hi

I've disabled and re-enabled the skill from Alexa and removed all devices that were Offline.
Now after re-enabling the skill, the devices are not discovered.

Any help?

Thanks

**Comment by hardillb** [comment]
Why did you disable the skill?
And where did you remove the devices?

You appear to have a valid oAuth token and a list of devices and the service returned that list of devices to Amazon correctly multiple times just after 7am (UK time) today.

**Comment by woody4165** [comment]
I was having issues with devices not responding, so I thought removing and reinstalling the skill was the correct thing to do.
The devices have been removed (forget) from the list of devices (offline) in the web app of Alexa 

Now Alexa skill is enabled
![schermata 2019-01-21 alle 10 36 34](https://user-images.githubusercontent.com/6501444/51465572-74822000-1d68-11e9-8e2c-3430b41f9e29.png)

but after searching the devices they don't appear in the list

![schermata 2019-01-21 alle 10 37 31](https://user-images.githubusercontent.com/6501444/51465654-9e3b4700-1d68-11e9-9175-9165c8eb358c.png)

**Comment by hardillb** [comment]
Does Alexa give any error when you trigger discovering devices?

I can only assume that because you told Alexa to forget the devices it is ignoring them.

In not sure how to undo that, you might have to talk to Amazon support

**Comment by woody4165** [comment]
No error, I will try to contact Amazon support. :-(

Thanks

**Comment by woody4165** [comment]
I've called Amazon support, but it was really hard to explain the situation, they don't understand the fact that within Node-Red there is only a definition of "something" and not a real device.

I've tried to create a new device with a new name, but this didn't work either, as well changing the NodeRed Alexa Skill password and reinstall the skill with the new credentials.

Is it possible to test with a new NodeRed Alexa Skill account, just to test if this can be the solution?

**Comment by woody4165** [comment]
Ok. I will keep in mind if I have to call them back.

I'll wait when you wipe my tokens and I'll try again.

Sorry if I have done this mess, can be a warn for others!

Thanks

**Comment by hardillb** [comment]
I've cleaned up all your oAuth tokens, please unlink and relink and see what happens

**Comment by woody4165** [comment]
Thanks.
Sorry, but my english is not so good, what do you mean by unlink and relink? 
The skill in the Alexa app?

**Comment by hardillb** [comment]
yes, disable the skill and re-enable it so it asks for the service username/password

**Comment by woody4165** [comment]
Yeah!
Thanks, it's working now.
Now I know I don't have to unlink the skill.
